### ISSUE TITLE
parse_from_txt not working with mode="coefficients"

### ISSUE THREAD

**Issue by LorenzoRinaldi** [it-is-me-mario/MARIO#47]
world = mario.parse_from_txt(f"{path}\\coefficients", table='SUT', mode="coefficients")

returns this error

runcell('Parse aggregated database from excel', 'C:/Users/loren/Documents/GitHub/SESAM/GT-IOA/2. Add_sectors.py')
Traceback (most recent call last):

  File "C:\Users\loren\Documents\GitHub\SESAM\GT-IOA\2. Add_sectors.py", line 23, in <module>
    world = mario.parse_from_txt(f"{pd.read_excel(paths, index_col=[0]).loc['Database',user]}\\a. Aggregated_SUT\\coefficients", table='SUT', mode="coefficients")

  File "C:\Users\loren\Documents\GitHub\SESAM\MARIO\mario\tools\parsersclass.py", line 76, in parse_from_txt
    matrices, indeces, units = txt_praser(path, table, mode)

  File "C:\Users\loren\Documents\GitHub\SESAM\MARIO\mario\tools\tableparser.py", line 318, in txt_praser
    read = all_file_reader(

  File "C:\Users\loren\Documents\GitHub\SESAM\MARIO\mario\tools\utilities.py", line 204, in all_file_reader
    readers(file, file)

  File "C:\Users\loren\Documents\GitHub\SESAM\MARIO\mario\tools\utilities.py", line 155, in readers
    read[key][inner_key] = pd.read_csv(

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\c_parser_wrapper.py", line 92, in __init__
    ) = self._extract_multi_indexer_columns(

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\base_parser.py", line 337, in _extract_multi_indexer_columns
    index_names, _, _ = self._clean_index_names(

  File "C:\Users\loren\anaconda3\lib\site-packages\pandas\io\parsers\base_parser.py", line 948, in _clean_index_names
    name = cp_cols[c]

IndexError: list index out of range

**Comment by mohammadamint** [comment]
seems to be an issue with the file itself

**Comment by nigolred** [comment]
I also checked the issue and got the same error.

Simply run this file

```
#%% Import any Exiobase database
import mario
m = 'pxp'
local_path = f'.../Exiobase 3.8.2/IOT/IOT_2019_{m}.zip'
World = mario.parse_exiobase_3(local_path)

#%%
path = r'...\Any folder\flows'
World.to_txt(path, coefficients=True, flows=False)
# %%
path = r'...\Any folder\coefficients'

World_new = mario.parse_from_txt(path, table='IOT', mode='coefficients')
# %%

```

I also got the error IndexError: list index out of range

Me and @LorenzoRinaldi tried to figure out what happens. It looks like there is an error in the way pandas read the first matrix (Y). Check readers(file, file) in MARIO\mario\tools\utilities.py. It is really strange cause the guide for assigning correct indeces and columns looks fine! 🤯

**Comment by mohammadamint** [comment]
Addressed through commit [4bfccaa]
